### ISSUE TITLE
tower-cli schedule create does not accept PATCH HTTP method

### ISSUE THREAD

**Issue by bp8017** [ansible/tower-cli#256]
Tower CLI 3.0.2

```
tower-cli schedule create --name "Example Schedule" --description "Example Schedule" --job-template 288 --rrule "DTSTART:20170302T110000Z RRULE:FREQ=HOURLY;INTERVAL=1" -u admin -p 'XXXXXXX' --force-on-exists

Error: The Tower server says you can't make a request with the PATCH method to that URL (https://127.0.0.1/api/v1/job_templates/288/schedules/7/).
```

**Comment by AlanCoding** [comment]
`/api/v1/job_templates/288/schedules/7/` looks like an invalid URL.

**Comment by bp8017** [comment]
It looks like the extra "--force-on-exist" does not work with the schedule create subcommand. However, it would be nice if it allowed for some level of idempotency by running it again with the same arguments as the other subcommands support.

**Comment by jangsutsr** [comment]
@bp8017 Thx for the bug report. It is fixed and will be shipped with 3.1.
